ethereumj-core/src/main/java/org/ethereum/samples/EventListenerSample.java

```diff
@@ -287,13 +283,8 @@ private void waitForEther(byte[] address, BigInteger requiredBalance) throws Int
      *  - Calls contract from 2 different addresses
      */
     private void deployContractAndTest() throws Exception {
-        ethereum.addListener(new EthereumListenerAdapter() {
-            // when block arrives look for our included transactions
-            @Override
-            public void onBlock(Block block, List<TransactionReceipt> receipts) {
-                EventListenerSample.this.onBlock(block, receipts);
-            }
-        });
+        // when block arrives look for our included transactions
+        ethereum.subscribe(to(BlockAdded.class, this::onBlock));
```

Comment:
I see old method usage in `private void replayOnly() throws Exception`
`ethereum.addListener(blockReplay);`